src/node_http_parser.cc

```diff
@@ -472,6 +472,9 @@ class Parser : public AsyncWrap {
   static void Consume(const FunctionCallbackInfo<Value>& args) {
     Parser* parser;
     ASSIGN_OR_RETURN_UNWRAP(&parser, args.Holder());
+
+    CHECK(args[0]->IsExternal());
+
```

Comment:
I was only talking about removing the empty line I left the comment at, not both of them. The idea was to make everything related to extracting `stream` from `args` one visual block, if my comment was too short to be clear :)
You are not obliged to do that though, and I'm fine with both of your variants too, that's just my personal preference you may or may not agree with.